src/SME.SGP.WorkerService/WorkerRabbitMQ.cs

```diff
@@ -104,12 +109,32 @@ private async Task TratarMensagem(BasicDeliverEventArgs ea)
                     {
                         SentrySdk.CaptureMessage($"RABBITMQ ERRO - {ea.RoutingKey}", SentryLevel.Error);
                         SentrySdk.CaptureException(ex);
+
+                        if (mensagemRabbit.NotificarErroUsuario)
+                            await NotificarErroUsuario(ex.Message, mensagemRabbit.UsuarioLogadoRF, ea.RoutingKey);
```

Comment:
acho melhor só passar a ex.Message se a exception for do tipo NegocioException, senão vamos mandar uma mensagem genérica.